<launch>
    <node pkg="arx_r5_controller" type="R5Controller" name="remote_master_l" output="screen">
        <param name="arm_control_type" value="remote_master_v1"/>
        <param name="arm_can_id" value="can0"/>
        <param name="arm_end_type" value="1"/>

        <param name="pub_topic_ee_name_v1" value="/ARX_VR_L"/>
        <param name="pub_topic_joint_name_v1" value="/joint_information"/>
    </node>

    <node pkg="arx_r5_controller" type="R5Controller" name="remote_master_r" output="screen">
        <param name="arm_control_type" value="remote_master_v1"/>
        <param name="arm_can_id" value="can2"/>
        <param name="arm_end_type" value="1"/>

        <param name="pub_topic_ee_name_v1" value="/ARX_VR_R"/>
        <param name="pub_topic_joint_name_v1" value="/joint_information2"/>
    </node>
</launch>